scripts/configs/obsparam_testfile.yaml

```diff
@@ -1,27 +1,47 @@
+# A healvis observation parameter file
+
+# --- IO Parameters ---
 filing:
   outdir: "./results/test_out"
   outfile_prefix: 'healvis'
   clobber: False
+
+# -- Simulation Parameters ---
+telescope:
+  config_dir: "./scripts/telescope_config"
+  array_layout: HERA65_layout.csv
+  telescope_config_name: HERA65_config.yaml
 freq:
   Nfreqs: 20
-  start_freq: 100000000.0
-  bandwidth:  30000000.0
-telescope:
-  array_layout: telescope_config/HERA65_layout.csv
-  telescope_config_name: telescope_config/HERA65_config.yaml
-sources:
-  catalog: 'gaussian_eor_shell'
-skymodel:
-    sigma: 0.031      # 31 mK
-    Nside: 64
-    savepath: 'flatspectrum.hdf5'
-    ref_chan: 10
+  start_freq: 100000000.0  # Hz
+  bandwidth:  100000000.0  # Hz
 time:
-  Ntimes: 5   # 24 hours
-  integration_time: 11.0
-  start_time: 2458098.5521759833
+  Ntimes: 5
+  integration_time: 100.0  # seconds
+  start_time: 2458098.5521759833  # Julian Date
+beam:
+  # options are ["airy", "gaussian", "<filepath>"]
+  beam_type: "airy"
+  # additional params for 'airy' or 'gaussian'
+  diameter: 15  # meters
+  gauss_width: 5.0  # degrees
+pols:
+  - 'xx'
+  - 'yy'
 select:
   bls: '[(0,11),(0,12),(11,12)]'
-
 fov: 110   # Deg
 Nskies: 1
+sources:
```

Comment:
This section was kept for compatibility with `pyuvsim` setup functions. If you're removing that dependency, we can probably get rid of it.